pkg/cmd/server/server.go

```diff
@@ -124,6 +132,10 @@ func NewCommand() *cobra.Command {
 	command.Flags().Var(logLevelFlag, "log-level", fmt.Sprintf("the level at which to log. Valid values are %s.", strings.Join(logLevelFlag.AllowedValues(), ", ")))
```

Comment:
Still need to switch these 2 flags (`plugin-dir`, `metrics-address`) to use the fields on the `serverConfig` struct. Then remove the unused local vars for these.